core-bundle/src/Command/ResizeImagesCommand.php

```diff
@@ -325,14 +329,30 @@ private function executeConcurrent(float $timeLimit, float $concurrent): int
             if (null === $process) {
                 continue;
             }
-            $this->finishSubProcess($process, $paths[$index]);
+
+            if (!$this->finishSubProcess($process, $paths[$index])) {
+                ++$failedCount;
+            }
+
             unset($processes[$index]);
             $this->updateOutput($processes, $counts, $paths);
         }
 
         $this->tableOutput->clear();
 
-        $this->io->writeln('Resized '.array_sum($counts).' Images.');
+        $count = array_sum($counts);
+
+        $this->io->writeln('Resized '.($count - $failedCount).' Images successfully.');
+
+        if ($failedCount > 0) {
+            $this->io->writeln('Resizing of '.$failedCount.' images failed.');
+        }
+
+        if (0 !== $failedCount && $count - $failedCount <= 0) {
+            $this->io->error('No image could be resized successfully.');
+
+            return 1;
+        }
```

Comment:
Shouldn't this condition be in line 345? If everything fails this will now output:
```
Resized 0 Images successfully.
Resizing of n images failed.
No image could be resized successfully.
```

Reply:
I think the output is OK this way. The error is formatted differently to the info messages:
![Bildschirmfoto 2020-08-24 um 15 36 58](https://user-images.githubusercontent.com/367169/91050980-bdbee380-e61f-11ea-8ec0-a809e45c383f.png)